e2e/cypress/integration/system_console/reporting/team_statistics_spec.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('System Console > Team Statistics', () => {
+    before(() => {
+        // # Visit team statistics page.
+        cy.visit('/admin_console/reporting/team_statistics').wait(TIMEOUTS.TWO_SEC);
+    });
+
+    it('MM-T906 Team Statistics displays expected content categories', () => {
+        // * Check that the header has loaded correctly and contains the expected text.
+        cy.get('.team-statistics__header span').should('be.visible').should('contain', 'Team Statistics for');
+
+        // * Check that the rows for the table were generated.
+        cy.get('.admin-console__content .row').should('have.length', 4);
+
+        // * Check that the title content for the stats is as expected.
+        cy.get('.admin-console__content .row').eq(0).find('.title').eq(0).should('contain', 'Total Active Users');
+        cy.get('.admin-console__content .row').eq(0).find('.title').eq(1).should('contain', 'Public Channels');
+        cy.get('.admin-console__content .row').eq(0).find('.title').eq(2).should('contain', 'Private Channels');
+        cy.get('.admin-console__content .row').eq(0).find('.title').eq(3).should('contain', 'Total Posts');
+        cy.get('.admin-console__content .row').eq(1).find('.title').eq(0).should('contain', 'Total Posts');
+        cy.get('.admin-console__content .row').eq(2).find('.title').eq(0).should('contain', 'Active Users With Posts');
+        cy.get('.admin-console__content .row').eq(3).find('.title').eq(0).should('contain', 'Recent Active Users');
+        cy.get('.admin-console__content .row').eq(3).find('.title').eq(1).should('contain', 'Newly Created Users');
+
+        // * Check that the values for the stats are valid.
+        cy.get('.admin-console__content .row').eq(0).find('.content').each((el) => {
+            cy.wrap(el).eq(0).invoke('text').then(parseFloat).should('be.gt', 0);
```

Comment:
Got failing test here. Maybe better if team stats to check is in a new team rather than eligendi.
<img width="944" alt="Screen Shot 2020-09-08 at 6 00 44 PM" src="https://user-images.githubusercontent.com/5334504/92462612-48493a80-f1fd-11ea-9380-6dc59d2fee50.png">